ts/Stock/StockToolsBindings.ts

```diff
@@ -750,26 +778,39 @@ var stockToolsBindings: Record<string, Highcharts.NavigationBindingsOptionsObjec
         className: 'highcharts-segment',
         // eslint-disable-next-line valid-jsdoc
         /** @ignore-option */
-        start: function (this: NavigationBindings, e: PointerEvent): Annotation {
+        start: function (
+            this: NavigationBindings,
+            e: PointerEvent
+        ): Annotation|void {
             var coords = this.chart.pointer.getCoordinates(e),
+                coordsX = this.getAssignedAxis(coords, 'x'),
+                coordsY = this.getAssignedAxis(coords, 'y'),
                 navigation = this.chart.options.navigation,
-                options = merge(
-                    {
-                        langKey: 'segment',
-                        type: 'crookedLine',
-                        typeOptions: {
-                            points: [{
-                                x: coords.xAxis[0].value,
-                                y: coords.yAxis[0].value
-                            }, {
-                                x: coords.xAxis[0].value,
-                                y: coords.yAxis[0].value
-                            }]
-                        }
-                    },
-                    navigation.annotationsOptions,
-                    (navigation.bindings as any).segment.annotationsOptions
-                );
+                options;
```

Comment:
is it necessary to declare the options here, instead of after the if statement, where the options var is assigned?
Additionally, why not `const`?

Reply:
It's a habit related to `var` being hoisted. I didn't change `var` to `let` or `const` because this ticket is not about refactoring. If I wrote it from scratch, I would definitively write it your way.